connector/mqtt/vernemq/README.md

```diff
@@ -0,0 +1,301 @@
+# MQTT 
+
+## Overview
+
+### Iot-Agent MQTT for Dojot
+
+The Dojot VerneMQ service is a extension of [VerneMQ](https://github.com/vernemq/vernemq) with some features for dojot case.
+
+![image](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/dojot/dojot/epic-100kMqttDevices/connector/mqtt/vernemq/docs/plant_uml/mqtt/diag_mqtt)
```

Comment:
You can see the image before merge here
http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/mprevide/dojot/readme_mqtt/connector/mqtt/vernemq/docs/plant_uml/mqtt/diag_mqtt